main.py

```diff
@@ -603,7 +608,15 @@ def handle_flows_install_delete(self, event):
             return
 
         force = bool(event.content.get("force", False))
+        if not flow_dict["flows"]:
+            log.error(f"Error, empty list of flows received. {flow_dict}")
+            return
+
         switch = self.controller.get_switch_by_dpid(dpid)
+        if not switch:
+            log.error(f"Switch, {switch}, was not found.")
```

Comment:
```suggestion
            log.error(f"Switch dpid {dpid} was not found.")
```

Trivial stuff, but can we have `dpid` interpolated here, otherwise `switch` would be None, not being much helpful in the log message.